editor.planx.uk/src/@planx/components/Question/Public/Question.tsx

```diff
@@ -33,12 +43,116 @@ export enum QuestionLayout {
 }
 
 const Question: React.FC<IQuestion> = (props) => {
-  const isUsingAltTheme = hasFeatureFlag("ALT_THEME");
+  const theme = useTheme();
+
+  const previousResponseId = props?.previouslySubmittedData?.answers?.[0];
+  const previousResponseKey = props.responses.find(
+    (response) => response.id === previousResponseId
+  )?.responseKey;
+
+  const formik = useFormik({
+    initialValues: {
+      selected: {
+        id: previousResponseId ?? "",
+        a: previousResponseKey ?? undefined,
+      },
+    },
+    onSubmit: (values) => {
+      setTimeout(
+        () => props.handleSubmit({ answers: [values.selected.id] }),
+        theme.transitions.duration.standard
+      );
+    },
+    validate: () => {},
+  });
+
+  let layout = QuestionLayout.Basic;
+  if (props.responses.find((r) => r.img && r.img.length)) {
+    layout = QuestionLayout.Images;
+  } else if (
+    props.responses.find((r) => r.description && r.description.length)
+  ) {
+    layout = QuestionLayout.Descriptions;
+  }
+
+  return (
+    <Card
+      handleSubmit={formik.handleSubmit}
+      isValid={Boolean(formik.values.selected.id)}
+    >
+      <QuestionHeader
+        title={props.text}
+        description={props.description}
+        info={props.info}
+        policyRef={props.policyRef}
+        howMeasured={props.howMeasured}
+        definitionImg={props.definitionImg}
+        img={props.img}
+      />
+      <FormControl sx={{ width: "100% " }}>
+        <FormHelperText style={visuallyHidden}>
+          {props.description ? DESCRIPTION_TEXT : ""}
+        </FormHelperText>
+        <FormLabel
+          style={visuallyHidden}
+          id={`radio-buttons-group-label-${props.id}`}
+        >
+          {props.text}
+        </FormLabel>
+        <RadioGroup
+          aria-labelledby={`radio-buttons-group-label-${props.id}`}
+          name={`radio-buttons-group-${props.id}`}
+        >
+          <Grid
+            container
+            spacing={layout === QuestionLayout.Basic ? 1 : 2}
+            alignItems="stretch"
+          >
+            {!props.text?.startsWith("Sorry") &&
+              props.responses?.map((response) => {
+                const onChange = () => {
+                  formik.setFieldValue("selected.id", response.id);
+                  formik.setFieldValue("selected.a", response.responseKey);
+                };
+                const buttonProps = {
+                  onChange,
+                };
 
-  return isUsingAltTheme ? (
-    <RadioQuestion {...props} />
-  ) : (
-    <ButtonQuestion {...props} />
```

Comment:
This file has a very noisy diff, but all that's happened here is that what was previously `RadioQuestion.tsx` is now just `Question.tsx` - there's no logical changes apart from the dropping of this ternary and the child becoming the parent.